app/index.html

```diff
@@ -27,27 +27,23 @@
     <!-- endbuild -->
 
     <!-- build:js static/scripts/angular.js -->
-    <script src="bower_components/angular/angular.js"></script>
-    <script src="bower_components/angular-route/angular-route.js"></script>
-    <script src="bower_components/angular-cookies/angular-cookies.js"></script>
-    <script src="bower_components/angular-truncate-2/src/angular-truncate-2.js"></script>
-    <script src="bower_components/angular-sanitize/angular-sanitize.js"></script>
+    <script src="bower_components/angular/angular.min.js"></script>
```

Comment:
I would not include minimised versions here, because it is harder to debug and the files are concatenated and minified in the build process anyway.